.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,35 @@
+name: Continuous integration
+on:
+  pull_request:
+  workflow_call:
+  workflow_dispatch:
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-version: ['7.4', '8.0', '8.1', '8.2', '8.3']
```

Comment:
I would have personally limited support to PHP 8.1 (and ideally 8.2), as older versions are [no longer supported](https://www.php.net/supported-versions.php).

However, the WordPress ecosystem is very focused on less technical people, which frequently use outdated shared hosting servers.

That's why WordPress supports very old PHP versions and recommends plugin maintainers do the same, otherwise you can find people having problems using your plugin.

Reply:
How much difference is there between PHP versions? I'm wondering how important it is to have each point release in the test matrix.

Reply:
Usually code written for v7.4 should work on newer versions, but this serves as a safety check to be completely sure about this.

At the end of the day, PHP does not follow SemVer, so there could be unexpected breaking changes. Not very usual, but having these checks is cheap, specially considering we don't have control over the envs the plugin will end up running.